### ISSUE TITLE
Pick Scrape no element in current MusicXML

### ISSUE THREAD

**Issue by snakebyte69** [w3c/musicxml#269]
Pick Scrape is a very common technique used by guitar players, there currently is no element for this.

**Comment by jsawruk** [comment]
You could fake it by using a glissando, but since pick scrape is not the same as a glissando, I think there should be a way to capture the semantics of a pick scrape.

**Comment by snakebyte69** [comment]
![pick scrape](https://user-images.githubusercontent.com/39090848/52456270-0c28a080-2ba8-11e9-84ca-c2b02980b9b4.jpg)

**Comment by mdgood** [comment]
This seems like something that could be added to issue #103 for guitar notation semantics. Or perhaps that issue should be split out into smaller issues like this? 

Even if we reserve full semantic text for MNX, perhaps we could do something like expand the features of the `<play>` element to get this working in a MusicXML context.

**Comment by snakebyte69** [comment]
Will this make it into MusicXML 4.0?

**Comment by dspreadbury** [comment]
No, it's not in the [MusicXML 4.0 milestone](https://github.com/w3c/musicxml/milestone/2).

**Comment by snakebyte69** [comment]
Is this going to make it into MusicXML 4.1?

**Comment by infojunkie** [comment]
Thanks for reviving this issue and related guitar ones. I'm working on Milestone 4.1 now, and it's my plan to include them. If I need some expert opinion on finer points of guitar notation, can I call upon you to help?
